### PR TITLE
RT: Fix terminus timeo

### PR DESCRIPTION
**Pull request by azime**

### CODE REVIEW

**Comment by antoine-de** [comment]
Jenkins, test this please

**Comment by azime** [comment]
Jenkins, test this please

File: source/time_tables/tests/departure_board_test_data.h [hunk 1]
```diff
@@ -60,20 +61,34 @@ struct departure_board_fixture {
...
+        b.data->compute_labels();
 
-        auto* sp_ptr = b.data->pt_data->stop_points_map["C:S0"];
+        sp_ptr = b.data->pt_data->stop_points_map["C:S0"];
         b.data->pt_data->codes.add(sp_ptr, "KisioDigital", "KisioDigital_C:S0");
         sp_ptr = b.data->pt_data->stop_points_map["C:S1"];
         b.data->pt_data->codes.add(sp_ptr, "KisioDigital", "KisioDigital_C:S1");
         b.data->pt_data->codes.add(sp_ptr, "AnotherSource", "AnotherSource_C:S1");
 
         sp_ptr = b.data->pt_data->stop_points_map["S43"];
```
Comment (pbougue) [inline source/time_tables/tests/departure_board_test_data.h:91]: I don't understand why you let this line.
Or maybe you should keep the test after?

Comment (pbougue) [inline source/time_tables/tests/departure_board_test_data.h:91]: My bad

**Pull request merged**